packages/astro/src/@types/astro.ts

```diff
@@ -586,6 +586,23 @@ export interface AstroUserConfig {
 		 * ```
 		 */
 		server?: string;
+		/**
+		 * @docs
+		 * @name build.assets
+		 * @type {string}
+		 * @default `'_astro'`
+		 * @description
```

Comment:
We should show when this feature was added:

```suggestion
		 * @version 2.0.0
		 * @description
```